app/src/main/cpp/skyline/services/pl/IPlatformServiceManager.cpp

```diff
@@ -30,4 +34,33 @@ namespace skyline::service::pl {
         response.copyHandles.push_back(handle);
         return {};
     }
+
+    Result IPlatformServiceManager::GetSharedFontInOrderOfPriority(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto languageCode{request.Pop<u64>()};
+
+        std::vector<u32> fontCodes;
+        std::vector<u32> fontOffsets;
+        std::vector<u32> fontSizes;
+
```

Comment:
Impl this properly probably, but otherwise use size_t instead of std::suze_t also use {} instead of = for definitions and {} instead of =0